src/select/src/components/SelectController.js

```diff
@@ -68,17 +69,18 @@ class SelectController extends Component<Props> {
   }
 
   composePropsWithComponents = (props :Object) :Object => {
+
     const { icon } = this.props;
+    const components :Object = {
+      Option: SelectOption,
+    };
 
     if (icon) {
-      const components = {
-        DropdownIndicator,
-      };
-
+      components.DropdownIndicator = DropdownIndicator;
       return { ...props, components };
```

Comment:
Can remove this line

Reply:
You mean `components.DropdownIndicator = DropdownIndicator;`?